packages/messaging/test/controller-get-token.test.ts

```diff
@@ -37,44 +38,47 @@ const ONE_DAY = 24 * 60 * 60 * 1000;
 describe('Firebase Messaging > *Controller.getToken()', () => {
   const sandbox = sinon.sandbox.create();
   const now = Date.now();
-  const expiredDate = new Date(now - ONE_DAY);
+  const expiredDate = now - ONE_DAY * 8; // 8 days ago
```

Comment:
i'm guessing the test wasn't supposed to be passing before this? Did you hit an issue when running the tests, or is the mocking making it work in either case?

Reply:
Yeah, the date in the test was wrong. But the way the date is created (using a Date object instead of a timestamp) was also wrong, which made the tests pass. :man_shrugging: